core/new-gui/src/app/workspace/service/execute-workflow/execute-workflow.service.ts

```diff
@@ -181,7 +165,7 @@ export class ExecuteWorkflowService {
     if (environment.amberEngineEnabled) {
       this.executeWorkflowAmberTexera();
     } else {
-      this.executeWorkflowOldTexera();
+      throw new Error('old texera engine not supported');
```

Comment:
can we document this change in the PR description?

Reply:
done